src/Adapter/Order/CommandHandler/UpdateProductInOrderHandler.php

```diff
@@ -112,6 +140,9 @@ public function handle(UpdateProductInOrderCommand $command)
         // Update quantity and amounts
         $order = $this->orderProductQuantityUpdater->update($order, $orderDetail, $product_quantity, $orderInvoice);
 
+        // Delete temporary specific prices
+        $this->clearTemporarySpecificPrices();
```

Comment:
Maybe this should be done after the hook call, so that specific prices are still in place when the module is executing its hook

Besides this call should be made in all cases, so we should have a try/catch/finally block that ensure it is executed because we don't want some roaming SpecificPrices in the nature ^^